### PR TITLE
dev-libs/hyperscan: fix python dependency, clean old, proxy co-maintainer

### PR DESCRIPTION
**Pull request by arkamar**
This PR fixes python dependencies in the ebuild. I also suggest to remove old versions and I would like to be a proxy co-maintainer of this package.

### CODE REVIEW

File: dev-libs/hyperscan/hyperscan-5.2.1.ebuild [hunk 1]
```diff
@@ -16,12 +16,14 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86"
 IUSE="+cpu_flags_x86_ssse3 static-libs"
 
-RDEPEND="${PYTHON_DEPS}
-	dev-libs/boost"
+RDEPEND="dev-libs/boost"
```
Comment (juippis) [inline dev-libs/hyperscan/hyperscan-5.2.1.ebuild]: Looks good, but this change requires a revbump to the package.

File: dev-libs/hyperscan/hyperscan-5.2.1.ebuild [hunk 2]
```diff
@@ -3,9 +3,9 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python2_7 )
+PYTHON_COMPAT=( python3_{6..8} )
 
-inherit cmake python-r1 flag-o-matic
+inherit cmake python-any-r1 flag-o-matic
```
Comment (juippis) [inline dev-libs/hyperscan/hyperscan-5.2.1.ebuild]: Please sort alphabetically where possible, these shouldn't cause conflicts.

Comment (croessner) [reply dev-libs/hyperscan/hyperscan-5.2.1.ebuild]: Python is not required here either.

Comment (arkamar) [reply dev-libs/hyperscan/hyperscan-5.2.1.ebuild]: Python is required as a build time dependency, see this https://github.com/intel/hyperscan/blob/d79973efb1fcf5ed338122882c1f896829767fb6/CMakeLists.txt#L75-L82

**Comment by arkamar** [comment]
Inherits are ordered and ebuild is revbumped.

**Comment by croessner** [comment]
Python is not required for building. Only if you added tests, which we do not do. Therefore python can be completely removed from hyperscan. I already tried it and it compiles fine without python

**Comment by arkamar** [comment]
My point is that hyperscan fails in configuration phase when python is not present. 
```cmake
if(PYTHONINTERP_FOUND)
    set(PYTHON ${PYTHON_EXECUTABLE})
else()
    message(FATAL_ERROR "No python interpreter found")
endif()
```
Therefore, it is necessary for configuration to succeed. The other point is that user can define `SOURCE_DATE_EPOCH` environment variable (for whatever reason) and then python is used for its processing.

If we do not want python dependency we should modify `CMakeLists.txt` appropriately. But I believe it would be better to propose those modification to the upstream and preserve python dependency until upstream accepts them.

**Comment by arkamar** [comment]
@juippis ping

**Comment by arkamar** [comment]
Thanks!

**Pull request closed without merging**